src/components/implementation/no_interface/cbboot/quarantine.c

```diff
@@ -401,14 +445,16 @@ fault_quarantine_handler(spdid_t spdid, long cspd_dspd, int cap_ccnt_dcnt, void
 
 		/* FIXME: which fork should be used here? This pulls the most
 		 * recent fork. */
-		f_spd = LAST_LIST(d, n, p)->fork_spd;
+		f_spd = 14; // LAST_LIST(d, n, p)->fork_spd;
 		printd("Fixing routing table after fork from %d -> %d\n",
 				d_spd, f_spd);
 
 		// TODO: add / change ucap, routing table
+		// should this do anything else???? Feel like it probably should
```

Comment:
Put yourself in the position of someone reading the code who doesn't know it.  What do you think they'd get from this comment.  Please ask yourself what you want to convey, and convey it directly and tersely.